mcx/sample.py

```diff
@@ -484,6 +485,8 @@ def get_unravel_fn():
 
     _, unravel_fn = get_unravel_fn()
 
+    init_rhat, update_rhat = online_gelman_rubin()
+    rhat_state = init_rhat(init_state.position.shape[1])
```

Comment:
`init_rhat(num_chains)`?

Reply:
`init_rhat` needs the number of variable not chains.

Reply:
You're right, my bad